### PR TITLE
SUPPORT-65378-andes-table

### PR DESCRIPTION
**Pull request by Sidahmeders**
Screenshot from 2023-01-02 12-22-55

### CODE REVIEW

File: src/Table/andestable.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    args: {
+        data,
+        columns: getFieldColumns,
+        SubComponent: ExpanderComponent,
+        pageSizeOptions: [5, 10, 20, 25, 50, 100],
+    },
+};
+
+const Template = args => <AndesTable {...args} />;
```
Comment (StoikovOleh) [inline src/Table/andestable.stories.jsx:59]: Add default template without custom components so we will have the default template first and second template with custom components

File: src/Table/andestable.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        },
+        loading: {
+            control: 'boolean',
+            ...getComponentPropType('boolean', false),
+        },
+        data: {
+            ...getComponentPropType('Array<any>'),
+        },
+        columns: {
+            ...getComponentPropType('Array<Column>'),
```
Comment (StoikovOleh) [inline src/Table/andestable.stories.jsx:42]: we need better documentation for the columns so user could see all parameters

Comment (StoikovOleh) [reply src/Table/andestable.stories.jsx:42]: https://github.com/TanStack/table/tree/v6#columns

Comment (Sidahmeders) [reply src/Table/andestable.stories.jsx]: please check again

**Review by StoikovOleh — APPROVED** [review]
merging to add my changes to fix code examples

**Pull request merged**